AUCapture-Console/AUCapture-Console.csproj

```diff
@@ -14,17 +14,12 @@
     <DebugType>none</DebugType>
   </PropertyGroup>
   <ItemGroup>
+  <PackageReference Include="Microsoft.Win32.Registry" Version="5.0.0" />
```

Comment:
Shouldn't this match tab level of shared memory? Might be my android client bugging, also OCD.

Reply:
It should, I just missed it when I copied it.